.github/workflows/main.yml

```diff
@@ -10,30 +10,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ["3.6", "3.7", "3.8", "3.9", "3.10"]
-        os: [ubuntu-latest, windows-latest]
+        python: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
Dropping Python 3.6 should be a separate PR I think.

Reply:
can we do this together? because setuptools_scm will fail for python 3.6, that was the reason to put it in this PR as well :grin:

Reply:
It is a known issue, but here is the fix: https://github.com/ESSS/pytest-regressions/blob/c2cfa9906a3df3474a7a45decc267389be941eb1/setup.py#L15-L18

Reply:
https://github.com/gabrielcnr/pytest-datadir/pull/42

Reply:
Merged